js/wp-seo-shortcode-plugin.js

```diff
@@ -0,0 +1,281 @@
+/* global console */
+/* global tinyMCE */
+/* global wpseoShortcodePluginL10n */
+/* global ajaxurl */
+/* global YoastSEO */
+/* global yoastShortcodePlugin */
+/* jshint -W097 */
```

Comment:
We shouldn't exclude jshint checks unless there is a very good reason to do so, both checks have very good reasoning why they are included, why are they excluded?

-W097: "Put 'use strict' in a function", this is to ensure that concatenation always works.
-W003: "Don't use a function before it's defined.", yes we have hoisting in javascript but following this rule makes for cleaner code.